sdk/python/feast/base_feature_view.py

```diff
@@ -32,6 +33,7 @@ def __init__(self, name: str, features: List[Feature]):
         self._name = name
         self._features = features
         self._projection = FeatureViewProjection.from_definition(self)
+        self.created_timestamp: Optional[datetime] = None
```

Comment:
This will need be written into the proto object in the `to_proto` method, and read out if it's present in the `from_proto` method; otherwise we'll lose this information when the feature view is persisted in the registry object.

Reply:
Hey @achals, we already [read](https://github.com/feast-dev/feast/blob/2435777d87c19d4eee670a7ccf5e73e88e22ce9d/sdk/python/feast/feature_view.py#L379) from and [write](https://github.com/feast-dev/feast/blob/2435777d87c19d4eee670a7ccf5e73e88e22ce9d/sdk/python/feast/feature_view.py#L300) to proto, right? But I forget to add the read/write in the ODFV. Is this your point? BTW, WDYT think about "promoting" `created_timestamp` field to the `spec` instead inside `meta`

Reply:
Ah, you're right. Thanks for addressing this.